Misc/NEWS.d/next/Core and Builtins/2023-09-24-22-26-56.gh-issue-109795.gI7KW6.rst

```diff
@@ -0,0 +1 @@
+:func:`!_thread.start_new_thread` now uses raw memory allocator to allocate new thread bootstate. This helps to avoid crashes in case when new thread gets started at interpreter finalization.
```

Comment:
Oh. Please remove this Changelog entry. I don't think that it's worth it. It doesn't impact directly users and it's a recent regression, no?